mlx-rs/src/ops/indexing/mod.rs

```diff
@@ -283,25 +293,25 @@ pub trait IndexMutOp<Idx, Val> {
     }
 }
 
-/// A marker trait for range bounds that are `i32`.
-pub trait IndexBounds: RangeBounds<i32> {}
+// /// A marker trait for range bounds that are `i32`.
```

Comment:
Shall we remove the commented out ones?

Reply:
Yeah, forgot to remove those lines. Resolved in [bfe8071](https://github.com/oxideai/mlx-rs/pull/122/commits/bfe80719e6306a96493372525ab49d6065d8cd2e)